R/TealAppDriver.R

```diff
@@ -365,16 +372,32 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
 
       if (identical(slices_suffix, "selection_manual")) {
         checkmate::assert_numeric(input, len = 2)
+        extra_formals <- formals(app$set_inputs)
+
+        dots <- rlang::list2(...)
+
+        checkmate::assert_choice(dots$priority_, formals(self$set_inputs)[["priority_"]])
```

Comment:
```suggestion
        checkmate::assert_choice(dots$priority_, formals(self$set_inputs)[["priority_"]], null.ok = TRUE)
```
dots$priority_ can be null.

Reply:
I think we should also check 'wait_' since we are using it here and not just passing it to the parent function.